tests/aggregation/tests.py

```diff
@@ -1004,6 +1004,10 @@ def test_annotate_over_annotate(self):
 
         self.assertEqual(author.sum_age, other_author.sum_age)
 
+    def test_same_annotation_aggregate_alias(self):
+        with self.assertRaisesMessage(ValueError, "Aggregation alias 'age_alias' overrides existing annotation"):
```

Comment:
Not a native speaker but wonder if _collides_ instead of _overrides_ would be more appropriate.

Reply:
It would have to be _collides with_, but perhaps _conflicts with_ would be better:

```suggestion
        msg = "Aggregation alias 'age_alias' conflicts with an existing annotation."
        with self.assertRaisesMessage(ValueError, msg):
```